docs/0_overview.md

```diff
@@ -0,0 +1,44 @@
+# Overview
+
+Crossplane is an open source control plane that allows you to manage
+infrastructure the Kubernetes way. It provides the following features:
+
+- Deployment and management of cloud provider managed services using the
+  Kubernetes API.
+- Management and scheduling of configuration data across multiple Kubernetes
+  clusters.
+- Separation of concern between [infrastructure owners](), [application
+  owners](), and [developers]().
+- Infrastructure agnostic packaging of applications and their dependencies.
+
+Crossplane does not:
+
+- Require that you run your workloads on Kubernetes.
+- Manage the data plane across Kubernetes clusters.
```

Comment:
Not as a first class managed resource, yet ... but I'd be surprised if this wan't part of  thisenvironment stack config in the future.

Reply:
Agreed, but I think it is very possible that we defer to other projects that we just install for you.